packages/compass-export-to-language/src/modules/transpiler.spec.ts

```diff
@@ -154,6 +154,7 @@ FindIterable<Document> result = collection.find(filter);`);
     });
 
     for (const protectConnectionStrings of [false, true]) {
+      // Security-relevant test -- description is in the protect-connection-strings e2e test.
```

Comment:
these are not used anywhere yet, right?

Reply:
Yup – just to clarify that a) these are relevant to security and b) already included in the report elsewhere